docs/documentation/aurora-config.md

```diff
@@ -648,6 +648,25 @@ logging:
   index: ""
 ```
 
+### Topology
+Support for organizing the topology view is available with the `topology` configuration.
+
+| Name                  | Default | Description                                                             |
+|-----------------------|---------|-------------------------------------------------------------------------|
+| `topology/partOf`     |         | Define the group the deployment belongs to.                             |
```

Comment:
Jeg er usikker på om denne beskrivelsen bør være litt tydeligere. Det kan virke for brukere at det er grupper som allerede eksisterer et sted. Så da lurer de nok på hvilke grupper det er snakk om. 
Kan du se om du klarer å få frem at det er selv definerte grupper?

Reply:
Jeg endret teksten og la til litt mer utfyllende beskrivelse under. Litt usikker på om det ble noe bedre?